canonical/charmhub.io
# Integrations tab – skeleton

## Done
Implement basic integrations tab (but hide it from the navigation)

## How to QA
- Visit https://charmhub-io-1408.demos.haus/openstack-dashboard/integrations
- See a list of integrations
- Ensure the heading is accurate about the number of integrations
- Search box should be there (but not work)

## Issue / Card
Fixes https://github.com/canonical/marketplace-tribe/issues/2841
Fixes https://github.com/canonical/marketplace-tribe/issues/2842

## Screenshots
![image](https://user-images.githubusercontent.com/479384/193820996-3f586b2b-d5c0-4302-ba39-bc8c6ffc7321.png)
